### DISCUSSION TITLE
Synced Lyrics

### DISCUSSION THREAD

**Discussion by DrasticShadow** [SimonIT/spotifylyrics#44] [Q&A] [answered]
Hi, there I've been using this app and am loving it, I can't seem to figure out if there's a way for the lyrics to change automatically and not have me manually scroll to read. I've turned on the synced lyrics option in the options bar but the lyrics still stay stagnant. An insight on this would be greatly appreciated. Thank you.

**Comment by SimonIT** [comment, marked as answer]
Hi thanks for using spotifylyrics!
Maybe you were just out of luck? If the program couldn't find the lyrics on any of the synchronized services, it fallbacks to a non synchronized service. 
You can verify it by searching manually on the linked pages in the readme that nothing is broken in the program and the lyrics for the song aren't just not available as synchronized
